main.py

```diff
@@ -147,25 +145,26 @@ def login(driver, username, password):
     password_input.send_keys(password)
 
     LOGGER.info("Logging in")
-    driver.find_element_by_xpath("//input[@value='LOG IN']").click()
-    wait_until_visible(driver=driver, xpath="//span[text()='My Account']")
-
+    driver.find_element_by_xpath("//input[@value='SIGN IN']").click()
+    wait_until_visible(driver=driver, xpath="//a[@data-path='myAccount:greeting']")
+    
     LOGGER.info("Successfully logged in")
 
 
-def select_shoe_size(driver, shoe_size):
-    LOGGER.info("Waiting for size dropdown button to become clickable")
-    wait_until_clickable(driver, class_name="size-dropdown-button-css", duration=10)
-
-    LOGGER.info("Clicking size dropdown button")
-    driver.find_element_by_class_name("size-dropdown-button-css").click()
-
+def select_shoe_size(driver, shoe_size, shoe_type):
     LOGGER.info("Waiting for size dropdown to appear")
-    wait_until_visible(driver, class_name="expanded", duration=10)
+    wait_until_visible(driver, class_name="size-grid-button", duration=10)
 
     LOGGER.info("Selecting size from dropdown")
-    driver.find_element_by_class_name("expanded").find_element_by_xpath(
-        "//button[text()='{}']".format(shoe_size)).click()
+
+    if shoe_type == "Y" or shoe_type == "C":
```

Comment:
`shoe_type in ("Y", "C")` ... this is a pet peeve of mine 😄

Reply:
No problem, i'll update this. Great overall work by the way! :)